src/ansys/fluent/core/session.py

```diff
@@ -374,10 +377,14 @@ def upload(
             Name of the local file to upload to the server.
         remote_file_name : str, optional
             remote file name, by default None
+        command_name : str, optional
```

Comment:
This function should not be made specific to a command name.

Reply:
We will find proper solution for this in future. Thank you.